packages/terra-icon/src/Icon.module.scss

```diff
@@ -21,3 +21,66 @@
     transform: scaleX(-1);
   }
 }
+
+:global {
+  /* stylelint-disable at-rule-no-vendor-prefix */
+  /* stylelint-disable property-no-vendor-prefix */
+  @-webkit-keyframes tui-spin-ltr {
```

Comment:
do we get this for free with autoprefixer? also can someone tell me if this is necessary at all? caniuse says it's supported by the browsers that we support https://caniuse.com/?search=%40keyframes

Reply:
removed all the prefixers here https://github.com/cerner/terra-core/pull/3187/commits/bf44dee3d4a6952d87319321968f0386aff790e0.